README.md

```diff
@@ -26,3 +26,6 @@ If this is true, then it also true that little-o implies big-o.
 The difference of the two definitions is that little-o uses < which makes the growth much stricter when
 compared to the definition for big-o. The definition for big-o uses a <= which makes the growth for g(x)
 less restrictive. This difference between the definitions is the reason why little-o can imply big-o.
+
+The other difference between the definitions is that the little-o definition requires the inequality to hold
+for all positive constants while the big-o definition allows the inequality to hold for the specific pair.
```

Comment:
What is "the specific pair"?

Reply:
The specific pair refers to some constant C and the value $n_0$.

Reply:
But the definition of $o$ doesn't require it to be true for all values of $n_0$.

Reply:
Yes, the definition of little-o states that there is at least one value of $n_0$ that the inequality holds for all $n \leq n_0$.

Reply:
Then I don't understand why you're referring to "the specific pair". This sounds like it would be a difference to $o$.

Reply:
Yes I should have phrased it such that the big-O definition allows the inequality to hold for all $n \geq n_{0}$ and for the constant C. So it is much less restrictive compared to the little-o definition.

Reply:
Ok, then please say that.